### DISCUSSION TITLE
graphql.error.base.GraphQLError

### DISCUSSION THREAD

**Discussion by Megidd** [saleor/saleor#9529] [Q&A] [answered]
My `saleor` is at version `3.0.0` and my `saleor-dashboard` is at `3.0.0`. When I access `saleor-dashboard`, the `saleor` throws these errors:

```bash
{"asctime": "2022-04-12T16:33:56Z", "levelname": "INFO", "lineno": 360, "message": "A query had an error", "name": "saleor.graphql.errors.handled", "pathname": "/home/m3/repos/saleor/./saleor/graphql/views.py", "process": 1289406, "threadName": "uWSGIWorker3Core0", "exc_info": "graphql.error.base.GraphQLError: Unknown type \"AppExtensionFilterInput\". Perhaps you meant \"CollectionFilterInput\", \"ExportFileFilterInput\", \"AppFilterInput\", \"AttributeFilterInput\" or \"ShippingZoneFilterInput\"?", "hostname": "beys"}
{"asctime": "2022-04-12T16:33:56Z", "levelname": "INFO", "lineno": 360, "message": "A query had an error", "name": "saleor.graphql.errors.handled", "pathname": "/home/m3/repos/saleor/./saleor/graphql/views.py", "process": 1289406, "threadName": "uWSGIWorker3Core0", "exc_info": "graphql.error.base.GraphQLError: Cannot query field \"appExtensions\" on type \"Query\".", "hostname": "beys"}
{"asctime": "2022-04-12T16:33:56Z", "levelname": "WARNING", "lineno": 224, "message": "Bad Request: /graphql/", "name": "django.request", "pathname": "/home/m3/repos/saleor/saleor-venv/lib64/python3.9/site-packages/django/utils/log.py", "process": 1289406, "threadName": "uWSGIWorker3Core0", "status_code": 400, "request": "<WSGIRequest: POST '/graphql/'>", "hostname": "beys"}
```

According to [this](https://stackoverflow.com/a/65590507/3405291), I guess:

> ... because the types set up in your graphQL schema are conflicting with the types you are trying to use when querying.

Does anyone know how I can resolve the issue?

# Update

I dropped the whole database and ran these commands again:

```bash
python3.9 manage.py migrate
python3.9 manage.py collectstatic --noinput
python3.9 manage.py populatedb
python3.9 manage.py createsuperuser
python3.9 manage.py runserver
```
But I still receive errors.

![Screenshot_20220412_215004](https://user-images.githubusercontent.com/17475482/163018641-47a6b33b-ea04-49f7-9c75-15ae324f06ee.png)

**Comment by karolkielecki** [comment, marked as answer]
Hi @Megidd, if you're currently using 3.0, please try with dash 3.0.1 and core 3.0.3. However, I recommend using the latest stable release. If that error appears again, let me know.

**Reply by Megidd** [reply]
Thanks @karolkielecki :smiley: 

The error got resolved by checking out both `saleor` and `saleor-dashboard` at tag/release `3.2.0` :heavy_check_mark:

**Reply by karolkielecki** [reply]
Great!
